Makefile

```diff
@@ -19,7 +19,7 @@ OUTPUT_DIR ?= output
 REGISTRY ?= quay.io/kubernetes-service-catalog/
 IMAGE ?= $(REGISTRY)minibroker
 TAG ?= canary
-IMAGE_PULL_POLICY ?= Always
+IMAGE_PULL_POLICY ?= Never
```

Comment:
Hmm. I can see how `IfNotPresent` would do the wrong thing (since it wouldn't get rebuilt images).  But if we do `Never` then we'd need to push it to the minikube VM right? Where does that happen? (It's been a while since I've looked at this, so I'm having issues paging all this back in…)

Reply:
```
(eval $(minikube docker-env); make image)
```

Maybe I can include a target in this PR to do just that.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/56/commits/41d95bf7e0849387e782b6a32bc857ec8bcf4a43.